board/safety/safety_hyundai.h

```diff
@@ -259,8 +259,8 @@ static int hyundai_tx_hook(CANPacket_t *to_send, bool longitudinal_allowed) {
   // FCA11: Block any potential actuation
   if (addr == 909) {
     int CR_VSM_DecCmd = GET_BYTE(to_send, 1);
-    int FCA_CmdAct = (GET_BYTE(to_send, 2) >> 5) & 1U;
```

Comment:
thanks to @sunnyhaibin for the find, this was wrong, it was getting `FCA_StopReq`

Reply:
why didn't the tests catch this?